filters/filters.go

```diff
@@ -38,10 +38,13 @@ type FilterContext interface {
 	// route.
 	Served() bool
```

Comment:
We don't need it, indeed. It should be only the internal flag. I'll add the deprecation documentation